src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -485,7 +485,7 @@ public static String convertToStandardDuration(Long timeInMilliseconds) {
      * @param pattern           the pattern of the date and time string
      * @return                  the parsed {@code LocalDateTime} object, or {@code null} if there are errors
      */
-    public static LocalDateTime parseLocalDateTime(String dateTimeString, String pattern) {
+    public static LocalDateTime parseLocalDateTimeForSessionsForm(String dateTimeString, String pattern) {
```

Comment:
Sorry, I think you took my previous comment a little too seriously! `parseLocalDateTimeForSessionsForm(String, String, String)` is a facade for `parseLocalDateTime(String, String)`, which is the method that does the actual datetime parsing. You'll have to rename this method back to `parseLocalDateTime` (but not the other one!)

Reply:
Oops, Sorry. I misunderstood. :sweat_smile: